opencsp/app/sofast/lib/SystemSofastFringe.py

```diff
@@ -327,15 +344,15 @@ def run_gray_levels_cal(
         on_processing: Callable = None,
         on_processed: Callable = None,
     ) -> None:
-        """Runs the projector-camera intensity calibration and stores the results in self.calibration_images and
-        self.calibration.
+        """Runs the projector-camera intensity calibration and stores the results in self._calibration_images_captured and
+        self._calibration.
 
         Params:
         -------
         calibration_class : type[ImageCalibrationAbstract]:
             The type of calibration to use.
         calibration_hdf5_path_name_ext : str, optional
-            The callback to execute when capturing is about to start. Default is None
+            The pathname of the output HDF5 file, does not save if None. Default is None

```

Comment:
Thanks for fixing my ~typo~ copy-pasto 😳

Reply:
Of course 🙂